Wrapping/Generators/SwigInterface/CMakeLists.txt

```diff
@@ -17,11 +17,15 @@ option(ITK_USE_SYSTEM_SWIG "Use system swig. If OFF, swig is built as an externa
 mark_as_advanced(ITK_USE_SYSTEM_SWIG)
 
 # Minimal swig version
-set(swig_version_min 3.0.0)
+if(WIN32)
+  set(swig_version_min 4.0.1)
+else()
+  set(swig_version_min 3.0.0)
```

Comment:
Can we universally move to swig version 4.0.1 for all platforms?

Otherwise, we need to test both versions and be careful not to use new features.  

I feel that we should be aggressively moving towards new swig 4.0.1 feature incorporation:

- Python Sphinx compatibility added for Doxygen comments.
- Support for Doxygen documentation comments which are parsed and converted into JavaDoc or PyDoc comments.
- C++11 STL containers added for Java, Python and Ruby.
- Improved support for parsing C++11 and C++14 code
- Python module overhaul by simplifying the generated code and turning most optimizations on by default.


Each of the above improvements in swig 4.0 will need to be avoided/delayed until we remove the dependence on swig_version_min 3.0.0.

Reply:
And since we bundle our own version of SWIG by default, it should not be much of a problem.

Reply:
Yes -- good points. I retained it for Linux because Ubuntu 18.04, e.g., ships with 3.0.12. Adding a patch to require 4.0.1 and disabling `ITK_USE_SYSTEM_SWIG` by default.